shared/router-v2/account-switcher/index.tsx

```diff
@@ -0,0 +1,143 @@
+import * as React from 'react'
+import * as Kb from '../../common-adapters'
+import * as Styles from '../../styles'
+import HeaderHOC from '../../common-adapters/header-hoc'
+import * as ConfigTypes from '../../constants/types/config'
+import {Props as HeaderHocProps} from '../../common-adapters/header-hoc/types'
+
+export type AccountRowItem = {
+  account: ConfigTypes.ConfiguredAccount
+  fullName: string
+}
+export type RowsProps = {
+  accountRows: Array<AccountRowItem>
+  onAddAccount: () => void
+  onCreateAccount: () => void
+  onSelectAccount: (username: string) => void
+}
+
+export type Props = {
+  fullname: string
+  onCancel: () => void
+  onProfileClick: () => void
+  username: string
+} & RowsProps &
+  HeaderHocProps
+
+const AccountRow = ({
+  entry,
+  onSelectAccount,
+}: {
+  entry: AccountRowItem
+  onSelectAccount: (username: string) => void
+}) => (
+  <Kb.NameWithIcon
+    clickType={onSelectAccount}
```

Comment:
i don't think this is valid

Reply:
I changed `nameWithIcon` so it would be.